src/uct/sm/mm/base/mm_iface.c

```diff
@@ -64,6 +64,17 @@ ucs_config_field_t uct_mm_iface_config_table[] = {
     {NULL}
 };
 
+ucs_config_field_t uct_mm_ext_iface_config_table[] = {
+  {"MM_", "", NULL, ucs_offsetof(uct_mm_ext_iface_config_t, super),
+   UCS_CONFIG_TYPE_TABLE(uct_mm_iface_config_table)},
+
+  {"ERROR_HANDLING", "off", "Enable error handling support",
```

Comment:
"Expose error handling support capability", UCS_CONFIG_TYPE_YES_NO

Reply:
should i rename it to "ERROR_HANDLING_ENABLED" to use UCS_CONFIG_TYPE_YES_NO?
Why yes/no is preferable over on/off?

Reply:
because it's there is no logic in UCT which is enabled or disabled, so y|n is more suitable

Reply:
fixed